controller/array_action/array_mediator_ds8k.py

```diff
@@ -29,6 +29,10 @@
 ERROR_CODE_VOLUME_NOT_FOUND_OR_ALREADY_PART_OF_CS_RELATIONSHIP = '00000013'
 MAX_VOLUME_LENGTH = 16
 
+FC_PERSISTENT_OPTION = "persistent"
```

Comment:
see [my previous comments](https://github.com/IBM/ibm-block-csi-driver/pull/173#discussion_r438202305) about those...

Reply:
solved in https://github.com/IBM/ibm-block-csi-driver/pull/201